src_c/math.c

```diff
@@ -4074,6 +4103,11 @@ math_disable_swizzling(pgVector *self, PyObject *_null)
 }
 
 static PyMethodDef _math_methods[] = {
+#if PY_VERSION_HEX < 0x03070000
+    {"clamp", math_clamp_wrapper, METH_VARARGS, DOC_PYGAMEMATHCLAMP},
+#else
+    {"clamp", math_clamp, METH_FASTCALL, DOC_PYGAMEMATHCLAMP},
```

Comment:
You may have to cast to PyCFunction here:
```suggestion
    {"clamp", (PyCFunction)math_clamp_wrapper, METH_VARARGS, DOC_PYGAMEMATHCLAMP},
#else
    {"clamp", (PyCFunction)math_clamp, METH_FASTCALL, DOC_PYGAMEMATHCLAMP},
```

Reply:
This of casting to PyCFunction is a compatibility thing. I've made many FASTCALL functions and all had to be cast to PyCFunction, no matter their type.